### PR TITLE
ss-43: Add cards

### PR DESCRIPTION
**Pull request by teslmik**
chrome_nFNzanhuRL

### CODE REVIEW

**Comment by nikita-remeslov** [comment]
Conflicts

File: frontend/src/bundles/common/components/card-total/styles.module.scss [hunk 1]
```diff
@@ -0,0 +1,64 @@
+:root {
+    --border-radius-1: 1px;
+    --border-radius-16: 16px;
+    --gap-7: 7px;
```
Comment (nikita-remeslov) [inline frontend/src/bundles/common/components/card-total/styles.module.scss:4]:
```suggestion
    --gap-7: 8px;
```

File: frontend/src/bundles/common/components/card-total/styles.module.scss [hunk 2]
```diff
@@ -0,0 +1,64 @@
+:root {
+    --border-radius-1: 1px;
```
Comment (nikita-remeslov) [inline frontend/src/bundles/common/components/card-total/styles.module.scss:2]: can we use some default value?
4px
8px
12px
16px

File: frontend/src/bundles/common/components/card-total/styles.module.scss [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    --border-radius-16: 16px;
+    --gap-7: 7px;
+}
+
+.card {
+    width: 229px;
```
Comment (nikita-remeslov) [inline frontend/src/bundles/common/components/card-total/styles.module.scss:8]: please round this values

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]

File: frontend/src/bundles/common/components/card-total/styles.module.scss (hunk 2 above)
Comment (teslmik) [reply frontend/src/bundles/common/components/card-total/styles.module.scss:2]: no border by default

File: frontend/src/bundles/common/components/card-total/styles.module.scss [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+    --drop-shadow: drop-shadow(0 2px 12px #386dff1f);
+}
+
+.card {
+    width: 230px;
+    height: 106px;
+    overflow: hidden;
+    font-family: var(--font-family);
+    font-style: normal;
+    border-radius: calc(var(--border-1) * 16);
```
Comment (nikita-remeslov) [inline frontend/src/bundles/common/components/card-total/styles.module.scss:13]: why do you use calc?

File: frontend/src/bundles/common/components/card-total/styles.module.scss [hunk 5]
```diff
@@ -0,0 +1,64 @@
+:root {
+    --border-1: 1px;
+    --g-8: 8px;
+    --drop-shadow: drop-shadow(0 2px 12px #386dff1f);
+}
+
+.card {
+    width: 230px;
+    height: 106px;
```
Comment (nikita-remeslov) [inline frontend/src/bundles/common/components/card-total/styles.module.scss:9]: please round this value

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]

**Comment by VItaliy-Kalenichenko** [comment]
Check how content overflow works on cards with spendee.com
image

**Comment by nikita-remeslov** [comment]
Conflicts

**Review by nikita-remeslov — APPROVED** [review]

**Pull request merged**